src/wrapped_radix_engine_toolkit/low_level/execution_summary/execution_summary.rs

```diff
@@ -0,0 +1,164 @@
+use crate::prelude::*;
+
+use radix_engine::types::indexmap::IndexMap;
+use radix_engine_common::types::ComponentAddress as ScryptoComponentAddress;
+
+use radix_engine_toolkit::transaction_types::ExecutionSummary as RetExecutionSummary;
+use radix_engine_toolkit::transaction_types::ResourceIndicator as RetResourceIndicator;
+
+/// A summary of the execution of the manifest and the information that helps
+/// wallets present the contents of a transaction.
+#[derive(Clone, Debug, PartialEq, Eq, uniffi::Record)]
+pub struct ExecutionSummary {
+    /// Addresses of accounts withdraws from in the manifest.
```

Comment:
`withdrawn` instead of `withdraws`?

Reply:
either: `addresses_of_account_withdrawn_from` or `addresses_of_account_withdrawals` 

withdrawn is the verb
withdrawal is the noun.

Hmm.... perhaps `addresses_of_account_withdrawn_from` is best? and then analogously: `addresses_of_account_deposits` -> `addresses_of_account_deposited_into` ?

Reply:
the combination of `withdrawn` and `deposited` sounds the best to me.